### PR TITLE
SPOI-11289 Change the property file to support user friendly properties

### PR DESCRIPTION
**Pull request by chaithu14**
@deepak-narkhede @yogidevendra Please review and merge.

### CODE REVIEW

**Comment by yogidevendra** [comment]
Updated the title for this JIRA. Please do the same for PR as well.

File: kinesis-to-s3/src/main/resources/META-INF/properties.xml [hunk 1]
```diff
@@ -20,37 +20,76 @@
...
-    <property>
-      <name>dt.operator.S3OutputModule.prop.outputDirectoryPath</name>
-      <value>OUTPUT_DIRECTORY_PATH</value>
-    </property>
+  <property>
+    <name>dt.operator.KinesisInput.prop.accessKey</name>
+    <value>${apex.app-param.accessKeyForKinesisInput}</value>
+  </property>
+  <property>
+    <name>apex.app-param.accessKeyForKinesisInput</name>
```
Comment (yogidevendra) [inline kinesis-to-s3/src/main/resources/META-INF/properties.xml]: Please rearrange the order for readability. 
Use of ${apex.app-param.accessKeyForKinesisInput} should be done after it is defined. 

Interchange the ordering for line 23-26 and line 27-30. 
Similar for all other properties.

File: kinesis-to-s3/src/site/conf/cluster-memory-conf.xml [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-    KIND, either express or implied.  See the License for the
-    specific language governing permissions and limitations
-    under the License.
-
--->
-
-<configuration>
-  <property>
-    <name>dt.operator.S3OutputModule.prop.maxS3UploadPartitions</name>
```
Comment (yogidevendra) [inline kinesis-to-s3/src/site/conf/cluster-memory-conf.xml]: Both these properties should move to properties.xml with the default value.

**Comment by chaithu14** [comment]
@yogidevendra Incorporated review comments and updated PR.

**Pull request merged**